### PR TITLE
Makefile nits

### PR DESCRIPTION
**Pull request by kolyshkin**
No functional change, just readability/maintainability improvements.

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -131,13 +153,16 @@ verify-dependencies: vendor
...
-	CGO_ENABLED=1 GOARCH=arm GOARM=7 CC=arm-linux-gnueabihf-gcc $(GO) build -buildmode=pie $(EXTRA_FLAGS) -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION} $(EXTRA_LDFLAGS)" -tags "$(BUILDTAGS)" -o runc-armhf .
-	CGO_ENABLED=1 GOARCH=arm64 CC=aarch64-linux-gnu-gcc $(GO) build -buildmode=pie $(EXTRA_FLAGS) -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION} $(EXTRA_LDFLAGS)" -tags "$(BUILDTAGS)" -o runc-arm64 .
-	CGO_ENABLED=1 GOARCH=ppc64le CC=powerpc64le-linux-gnu-gcc $(GO) build -buildmode=pie $(EXTRA_FLAGS) -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION} $(EXTRA_LDFLAGS)" -tags "$(BUILDTAGS)" -o runc-ppc64le .
+	CGO_ENABLED=1 GOARCH=arm GOARM=6 CC=arm-linux-gnueabi-gcc $(GO_BUILD) -o runc-armel .
+	CGO_ENABLED=1 GOARCH=arm GOARM=7 CC=arm-linux-gnueabihf-gcc $(GO_BUILD) -o runc-armhf .
+	CGO_ENABLED=1 GOARCH=arm64 CC=aarch64-linux-gnu-gcc $(GO_BUILD) -o runc-arm64 .
+	CGO_ENABLED=1 GOARCH=ppc64le CC=powerpc64le-linux-gnu-gcc $(GO_BUILD)  -o runc-ppc64le .
```
Comment (thaJeztah) [inline Makefile:165]: Tempting to align these to make it visually easier to see the differences, but guess that's personal;

```suggestion
	CGO_ENABLED=1 GOARCH=arm GOARM=6 CC=arm-linux-gnueabi-gcc     $(GO_BUILD) -o runc-armel .
	CGO_ENABLED=1 GOARCH=arm GOARM=7 CC=arm-linux-gnueabihf-gcc   $(GO_BUILD) -o runc-armhf .
	CGO_ENABLED=1 GOARCH=arm64       CC=aarch64-linux-gnu-gcc     $(GO_BUILD) -o runc-arm64 .
	CGO_ENABLED=1 GOARCH=ppc64le     CC=powerpc64le-linux-gnu-gcc $(GO_BUILD) -o runc-ppc64le .
```

Comment (kolyshkin) [reply Makefile]: good idea, done.

**Comment by kolyshkin** [comment]
I've added `-mod=vendor` to `go build` flags so our CI won't have to download all the modules again and again.

**Comment by mrunalp** [comment]
LGTM

**Comment by AkihiroSuda** [comment]
LGTM

**Pull request merged**